apps/davi/src/Modules/Guilds/pages/CreateDiscussion.tsx

```diff
@@ -69,16 +70,18 @@ const CreateDiscussionPage: React.FC = () => {
     });
   }, [user, orbis]);
 
+  const [html, onHTMLChange] = useLocalStorageWithExpiry<string>(
+    `${guildId}/create-discussion/html`,

```

Comment:
Should the `create-discussion` be replaced by a more dynamic property? It might be easier for the proposal page to fetch the right discussion as well

Reply:
@Kenny-Gin1 Is there a scenario we need to keep multiple draft versions of discussions? And in the proposal creation, we only utilize the discussion id, right.. Any thoughts?

Reply:
We utilise the `discussionId` to fetch from orbis. But if that would API call fails, then I think the proper behavior would be for the proposal page to fetch from the `localStorage`